services/quaranteam/Dnd/.gitignore

```diff
@@ -0,0 +1,32 @@
+HELP.md
```

Comment:
what is this HELP.md file?

Reply:
This gitignore file was a part of the springboot file that was generated and I realize it is unnecessary.